circuits/ts/__tests__/CeremonyParams.test.ts

```diff
@@ -186,14 +188,14 @@ describe("Ceremony param tests", function () {
       });
     });
 
-    describe("TallyVotes circuit", function () {
+    describe("TallyVotes circuit", function test() {
       this.timeout(900000);
 
-      let circuit: tester.WasmTester;
+      let testCircuit: tester.WasmTester;
```

Comment:
Saw you changed here but not in other files, let's keep them all the same?

Reply:
I changed this one because it conflicts with `circuit` const which is declared above.

Reply:
Oh true